Microsoft.Toolkit.Uwp/Helpers/PrintHelper/PrintHelper.cs

```diff
@@ -424,7 +429,10 @@ private void CreatePrintPreviewPages(object sender, PaginateEventArgs e)
         private void GetPrintPreviewPage(object sender, GetPreviewPageEventArgs e)
         {
             PrintDocument printDoc = (PrintDocument)sender;
-            printDoc.SetPreviewPage(e.PageNumber, _printPreviewPages[e.PageNumber - 1]);
+            if (_printPreviewPages.Count() > 0)
+            {
+                printDoc.SetPreviewPage(e.PageNumber, _printPreviewPages[e.PageNumber - 1]);
```

Comment:
I went to hit the 'Print' button (in Dark Theme) and saw a blank page.  I think hit the 'Direct Print' button and saw the images.  When I closed the print window though, I hit an exception.  On an insider preview build.  I'll try on my RS5 machine next.

Reply:
There is already a PR for the Dark Theme issue. I could test the solution in this PR.
https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/2798

Reply:
@lucaasrojas thanks I know, I was more concerned about the crash.  I'll test this change in light mode.